package.json

```diff
@@ -128,9 +128,10 @@
     "yarn-run-all": "^3.1.1"
   },
   "dependencies": {
-    "@emotion/core": "^10.0.35",
-    "@emotion/styled": "^10.0.27",
+    "@emotion/core": "10.0.35",
+    "@emotion/styled": "10.0.27",
     "@react-aria/focus": "^3.0.2",
+    "emotion-theming": "10.0.27",
```

Comment:
We will have to update our Usage documentation to note that all applications using `radiance-ui` need to set up a `ThemeProvider` at the top level of their App--or at least at the highest-level that makes use of `radiance-ui` components: https://emotion.sh/docs/theming